### DISCUSSION TITLE
How Do Run HTML/ Frontend Live Server?

### DISCUSSION THREAD

**Discussion by AbhayGhante** [coder/code-server#5791] [Q&A]
Hello Evreyone!
I have installed code server on Umbrel(Cloud Virtual Machine/VPS (Azure); One click install :))
But unfortunatly; I am not able to run live server for html. Please someone help how To Run Live Server For Frontend Development.

**Comment by jsjoeio** [comment]
Hmm 🤔 I haven't used it in a while. Can you provide steps to reproduce? Then I can try locally and help you find a solution!
